integration-tests/utils/integration-utils.js

```diff
@@ -346,7 +353,7 @@ const withPageLogging = fn => async options => {
   return await fn(options)
 }
 
-const waitUntilLoadedOrTimeout = timeout => ({ waitUntil: ['load', 'domcontentloaded', 'networkidle0'], timeout })
+const waitUntilNetworkIdleOrTimeout = timeout => ({ waitUntil: ['networkidle0'], timeout })
```

Comment:
I'm renaming function accordingly since removing `load` and `domcontentloaded` options,